network/builder/src/dummy.rs

```diff
@@ -136,13 +141,16 @@ pub fn setup_network() -> DummyNetwork {
 
     let authentication_mode = AuthenticationMode::Mutual(listener_identity_private_key);
 
+    let network_context = Arc::new(NetworkContext::new(
+        network_id.clone(),
+        RoleType::Validator,
+        listener_peer_id,
+    ));
     // Set up the listener network
```

Comment:
comment should be above the assignment.

Reply:
Put a space between the preceding lines and the comment.  I actually did intend to associate it with the builder and not the network_context